src/components/LogOut/index.tsx

```diff
@@ -1,22 +1,31 @@
 import LocalStorage from '../../local-storage'
 import { useHistory } from 'react-router-dom'
+import { Button } from 'react-bootstrap'
 
 const LogOut = () => {
   const history = useHistory()
-  const deleteTokenFromLocalStorage = async () => {
-    await LocalStorage.removeToken()
-    await history.push('/')
+  const deleteTokenFromLocalStorage = () => {
+    LocalStorage.removeToken()
+    history.push('/')
+  }
+
+  const styles = {
+    customButton: {
+      backgroundColor: '#6f42c1',
+      border: 'none',
+      color: '#fff',
+      borderRadius: '4px',
+    },
   }
```

Comment:
You can move this outside the component as well

Reply:
Done.